tests/data/group_services/group_sssd/rule_sssd_offline_cred_expiration/comment.fail.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+
+SSSD_CONF="/etc/sssd/sssd.conf"
```

Comment:
Please move the setup boilerplate to function that you can source and call? It's cleaner that way.